pkg/kubelet/util/pluginwatcher/example_plugin.go

```diff
@@ -70,11 +70,6 @@ func (s *pluginServiceV1Beta2) RegisterService() {
 	v1beta2.RegisterExampleServer(s.server.grpcServer, s)
 }
 
-// NewExamplePlugin returns an initialized examplePlugin instance
```

Comment:
Why deleting the constructor?

Reply:
cause no callers for this constructor, only the NewTestExamplePlugin is used.